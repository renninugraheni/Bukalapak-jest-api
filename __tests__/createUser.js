const epCreateUser = require('../endpoints/epPostUser');
const body = require('../data/dataCreateUser');

describe('test api post user', () => {
    test('test create user', async () => {
        const response = await epCreateUser.postListUsers(body.bodyCreateUser);
        expect(response.status).toEqual(201);
        expect(response.body).toMatchSnapshot();
    });
});