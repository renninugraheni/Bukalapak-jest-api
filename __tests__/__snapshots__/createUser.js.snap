// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test api post user test create user 1`] = `
Object {
  "createdAt": "2020-02-17T11:09:08.983Z",
  "id": "209",
  "job": "leader",
  "name": "morpheus",
}
`;
